language/regexp/character_classes_spec.rb

```diff
@@ -96,9 +96,11 @@
     "\x7F".match(/[[:ascii:]]/).to_a.should == ["\x7F"]
   end
 
-  it "doesn't match non-ASCII characters with [[:ascii:]]" do
-    ('/[[:ascii:]]/').match("\u{80}").should be_nil
-    ('/[[:ascii:]]/').match("\u{9898}").should be_nil
+  not_compliant_on :opal do
+    it "doesn't match non-ASCII characters with [[:ascii:]]" do
+      ('/[[:ascii:]]/').match("\u{80}").should be_nil
+      ('/[[:ascii:]]/').match("\u{9898}").should be_nil
+    end
   end
```

Comment:
Why not use a tag/exclude in this case since the whole example is not compliant?